config.toml

```diff
@@ -59,7 +59,7 @@ disablePathToLower = true
     parent = "docs"
     weight = 4
     identifier = "camel-k"
-    url = "/camel-k/latest/"
+    url = "/projects/camel-k/"
```

Comment:
This needs to remain the same, the `/camel-k/latest/` is built by Antora, not fro the Hugo markdown file. As can be seen [on staging site](https://camel.apache.org/staging/camel-k/latest/).

Reply:
sure Mr.Zoran I`ll do changes and again do a pull request.